mass: 1.5
hover_thrust: 0.402
takeoff_height: 1.3
ctrl_hz: 100
odomTopicName: /gt_iris_base_link_imu

boundings:
  T_max: 20
  T_min: 2
  wx_max: 1.5
  wx_min: -1.5
  wy_max: 1.5
  wy_min: -1.5
  wz_max: 1.0
  wz_min: -1.0

cost:
  cost_px: 100
  cost_py: 100
  cost_pz: 100
  cost_qw: 100
  cost_qx: 100
  cost_qy: 100
  cost_qz: 100
  cost_vx: 30
  cost_vy: 30
  cost_vz: 30
  cost_thrust: 30
  cost_wx: 30
  cost_wy: 30
  cost_wz: 30